modules/shared/components/molecules/CreatePostHeader/CreatePostHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { ICreatePostHeader } from './ICreatePostHeader';
+import Avatar from '../../atoms/Avatar/Avatar';
+import TabGroup from '../TabGroup/TabGroup';
+
+const CreatePostHeader: FC<ICreatePostHeader.IProps> = (
+  props,
+): ReactElement => {
+  const { tabsData, checkedValue, changeValHandler, profilePic } = props;
```

Comment:
To consistent through the project we destructure the `props` in the function parameter

Reply:
if I destructured the props in the function parameter it's gonna take more lines in the file
and i'm following this pattern through almost every component i took before :joy: :joy: